apps/crn-auth-frontend/package.json

```diff
@@ -5,24 +5,27 @@
   "engines": {
     "node": "18.x"
   },
+  "type": "module",
   "scripts": {
     "start": "cross-env PORT=3002 TSC_COMPILE_ON_ERROR=true react-scripts start",
-    "build": "node scripts/build.js",
+    "build": "vite build",
     "typecheck": "tsc",
-    "test": "echo Run the workspace root script instead",
-    "eject": "react-scripts eject"
+    "test": "echo Run the workspace root script instead"
   },
   "dependencies": {
     "@asap-hub/auth": "workspace:*",
     "@asap-hub/auth-frontend-utils": "workspace:*",
     "@asap-hub/react-components": "workspace:*",
+    "@babel/preset-react": "7.23.3",
     "@babel/runtime-corejs3": "7.23.4",
+    "@vitejs/plugin-react": "^4.2.0",
     "history": "4.10.1",
     "path": "npm:path-browserify@1.0.1",
     "react": "17.0.2",
     "react-app-polyfill": "3.0.0",
     "react-dom": "17.0.2",
-    "react-router-dom": "5.3.4"
+    "react-router-dom": "5.3.4",
+    "vite": "^5.0.2"
```

Comment:
Should these be dependencies or dev dependencies?

Reply:
regular dependencies according to docs